Changelog

```diff
@@ -1,4 +1,83 @@
+    - FEATURE: Show startup progress dialog (glassez)
+    - FEATURE: Add keyboard shortcut to Download URL dialog (Chocobo1)
+    - FEATURE: Add ability to run external program on torrent added (glassez)
+    - FEATURE: Add infohash and download path columns (tristanleboss)
+    - FEATURE: Allow to set torrent stop condition (glassez, thalieht)
+    - FEATURE: Add a "Moving" status filter (tristanleboss)
+    - FEATURE: Introduce color palettes for both dark, light themes (Chocobo1)
+    - FEATURE: Add a "Use proxy for hostname lookup" option (Nathan Lewis)
+    - FEATURE: Introduce a 'change listen port' cmd option (BallsOfSpaghetti)
+    - BUGFIX: Handle .!qB extension more transparently (glassez)
+    - BUGFIX: Store hybrid torrents using "torrent ID" as basename (glassez)
+    - BUGFIX: Allow to open incomplete files from content view (glassez)
+    - BUGFIX: Enable Combobox editor for the "Mixed" file download priority (Aleksandr Cupacenko)
+    - BUGFIX: Enable shortcut folders for the Open and Save directory dialogs (Aleksandr Cupacenko)
+    - BUGFIX: Rename content tab "Size" column to "Total Size" (Aleksandr Cupacenko)
+    - BUGFIX: Fix scrolling to the lowermost visible torrent (Aleksandr Cupacenko)
+    - BUGFIX: Allow setting priority for partially downloaded multi-file torrent (An0n)
+    - BUGFIX: Focus save path when Manual mode is selected initially (Aleksandr Cupacenko)
+    - BUGFIX: Disable force reannounce when it is not possible (An0n)
+    - BUGFIX: Add horizontal scrolling for tracker list and torrent content (NotTsunami)
+    - BUGFIX: Enlarge "speed limits" icons (Chocobo1)
+    - BUGFIX: Change Downloaded to Times Downloaded in trackers tab (An0n)
+    - BUGFIX: Remove artificial max limits from "Torrent Queueing" related options (Chocobo1)
+    - BUGFIX: Preserve "skip hash check" when there is no metadata (glassez)
+    - BUGFIX: Fix DHT/PeX/LSD status when it is globally disabled (Kacper Michajłow)
+    - BUGFIX: Fix rate calculation when interval is too low (glassez)
+    - BUGFIX: Remove option of using icons from system theme (Nowshed H. Imran)
+    - BUGFIX: Add tooltip message when system tray icon isn't available (Chocobo1)
+    - BUGFIX: Improve sender field in mail notifications (Dmitry Vodopyanov)
+    - BUGFIX: Fix "Add torrent dialog" spill-over on smaller screens (Chocobo1)
+    - BUGFIX: Fix peer count issue when tracker responds with zero figure (summer)
+    - BUGFIX: Don't merge trackers by default (glassez)
+    - BUGFIX: Don't inhibit system sleep/auto shutdown for torrents stuck at downloading metadata (summer)
+    - BUGFIX: Allow to pause a checking torrent from context menu (summer)
+    - BUGFIX: Allow to use subnet notation in reverse proxy list (Chocobo1)
+    - BUGFIX: Fine tune translations loading for Chinese locales (sledgehammer999)
+    - BUGFIX: Fix torrent content checkboxes not updated properly (Chocobo1)
+    - BUGFIX: Correctly load state of use download folder (glassez)
```

Comment:
Maybe add something about the fact that it is about "watched folder" settings?

Reply:
OK